.github/workflows/release.yml

```diff
@@ -39,61 +39,23 @@ jobs:
           files: |
             dist/artifacts/kine-*
             dist/artifacts/*.txt
-
-      - name: "Read Vault secrets"
-        if: github.repository_owner == 'k3s-io'
-        uses: rancher-eio/read-vault-secrets@main
-        with:
-          secrets: |
-            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials username | DOCKER_USERNAME ;
-            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials password | DOCKER_TOKEN ;
-    
-      - name: Login to DockerHub with Rancher Secrets
-        if: github.repository_owner == 'k3s-io'
-        uses: docker/login-action@v3
-        with:
-          username: ${{ env.DOCKER_USERNAME }}
-          password: ${{ env.DOCKER_TOKEN }}
-
-      # For forks, setup DockerHub login with GHA secrets
-      - name: Login to DockerHub with GHA Secrets
-        if: github.repository_owner != 'k3s-io'
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_TOKEN }}
       
-      - name: Login to GitHub Container Registry
+      - name: Login to GitHub Container Registry (loft-sh)
+        if: github.repository_owner == 'loft-sh'
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
-          username: ${{ github.repository_owner }}
+          username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build and push (k3s-io)
-        if: github.repository_owner == 'k3s-io'
-        uses: docker/build-push-action@v6
-        with:
-          platforms: linux/amd64,linux/arm64,linux/arm/v7
-          context: .
-          target: multi-arch-package
-          push: true
-          tags: |
-            rancher/kine:latest
-            rancher/kine:${{ github.ref_name }}
-            ghcr.io/${{ github.repository_owner }}/kine:latest
-            ghcr.io/${{ github.repository_owner }}/kine:${{ github.ref_name }}
-
-      - name: Build and push (forks)
-        if: github.repository_owner != 'k3s-io'
+      - name: Build and push (loft-sh)
+        if: github.repository_owner == 'loft-sh'
         uses: docker/build-push-action@v6
         with:
           platforms: linux/amd64,linux/arm64,linux/arm/v7
           context: .
           target: multi-arch-package
           push: true
           tags: |
-            ${{ secrets.DOCKER_USERNAME }}/kine:latest
-            ${{ secrets.DOCKER_USERNAME }}/kine:${{ github.ref_name }}
             ghcr.io/${{ github.repository_owner }}/kine:latest
             ghcr.io/${{ github.repository_owner }}/kine:${{ github.ref_name }}
```

Comment:
@pascalbreuninger Would it make sense to add some release tag to indicate the AWS support has been added to this version?